Stack/Opc.Ua.Core/Types/Encoders/JsonEncoder.cs

```diff
@@ -499,8 +501,65 @@ public void PopNamespace()
         }
 
         private static readonly char[] m_specialChars = new char[] { s_quotation, s_backslash, '\n', '\r', '\t', '\b', '\f', };
-        private static readonly char[] m_substitution = new char[] { s_quotation, s_backslash, 'n', 'r', 't', 'b', 'f' };
+#if NETCOREAPP2_1_OR_GREATER
```

Comment:
In libraries NETSTANDARD2_1_OR_GREATER it is.

Reply:
and || NET5_0_OR_GREATER

Reply:
checking at the support for Span<T>, the condition i think would be NETSTANDARD2_1_OR_GREATER || NET5_0_OR_GREATER ?

https://learn.microsoft.com/en-us/dotnet/api/system.span-1?view=net-8.0#applies-to

Reply:
updated to NETSTANDARD2_1_OR_GREATER || NET5_0_OR_GREATER  as discussed